sw/device/lib/dif/autogen/dif_lc_ctrl_autogen_unittest.cc

```diff
@@ -0,0 +1,11 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// This file is auto-generated.
+
+#include "sw/device/lib/dif/dif_lc_ctrl_unittest.h"
+
+namespace dif_lc_ctrl_unittest {
+namespace {}  // namespace
+}  // namespace dif_lc_ctrl_unittest
```

Comment:
This is a liiiiitle bit silly. I know lc_ctrl is the only pathological one in this case. We can leave as-is, but it does bother me slightly, lol.

Reply:
Sorry I should have been more clear here besides just mentioning this in #8142: _this is only temporary_. In the near future, this file (and the autogen'd C file) will _not_ be empty and will contain autogenerated code / unit tests for **alerts**. It is just empty for now to put the file structure in place so that when the alert DIFs are autogenerated, not much integration work will have to be done. Does this make sense? Also, note other IPs that do not raise interrupts will be similar. If an IP does not generate interrupts nor alerts, then this would not exist (but I have not come across an IP for which that is the case yet).